core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -1245,8 +1246,8 @@ private[log] class CleanedTransactionMetadata {
    *
    * @param abortedTransactions The new found aborted transactions to add
    */
-  def addAbortedTransactions(abortedTransactions: List[AbortedTxn]): Unit = {
-    this.abortedTransactions ++= abortedTransactions
+  def addAbortedTransactions(abortedTransactions: util.List[AbortedTxn]): Unit = {
+    abortedTransactions.forEach(abortedTxn => this.abortedTransactions += abortedTxn)
```

Comment:
Can this be `this.abortedTransactions ++= abortedTransactions.asScala`? Adding in bulk is usually more efficient.

Reply:
Wanted to avoid scala conversion.